### ISSUE TITLE
Loop added to responsive?

### ISSUE THREAD

**Issue by kevadamson** [ganlanyuan/tiny-slider#412]
Is it possible for you to add loop as an option for responsive?
```
var slider = tns({
    loop: false
    responsive: {
        640: {
            loop: true
        }
    }
}); 
```

**Comment by denisyatsko** [comment]
No, see the Doc. But if it will be added, it's will be really good
https://github.com/ganlanyuan/tiny-slider#responsive-options

**Comment by kevadamson** [comment]
Yeah. When I disable using responsive on a slider that has a loop, it is causing issues. The hidden elements used for loop still exist when I inspect the code.
